content/applications/productivity/documents.rst

```diff
@@ -251,16 +246,26 @@ Upload <documents/upload>` files to it.
 
 .. _documents/upload:
 
-Upload by email
-^^^^^^^^^^^^^^^
+Files upload
+^^^^^^^^^^^^
+
+Switch on the :guilabel:`Allow upload` button to allow users to upload files from your workspace.
+
+Tick the :guilabel:`Upload by Email` checkbox to enable users to upload documents sent by email in
+the workspace.
+
+.. note::
+
+   Your **email server** must be configured to be able to create an **email alias**. If it isn't
+   configured yet:
 
-Select the :guilabel:`Download and Upload` option to enable users to upload their files to your
-workspace using an :guilabel:`Email Alias`. To create the email alias, enter its name in the
-:guilabel:`Email Alias` field. The :doc:`domain name <../general/email_communication/email_servers>`
-should be set by default, but you can modify it by clicking it.
+   #. Click :guilabel:`Choose or Configure Email Servers`;
+   #. In the :guilabel:`General Settings` page, scroll down to the :guilabel:`Discuss` section and
+      go to the :guilabel:`Alias Domain`;
+   #. Enter your email server details and :guilabel:`save`.
 
-The documents sent to this email alias are uploaded to the workspace using the chosen
-:ref:`tags <documents/tags>`.
+Enter an alias in the :guilabel:`Email Alias` field. All documents sent to this email address are
+uploaded to the workspace using the chosen tags.
```

Comment:
I think this is actually described (and to a greater extent since this also depends on the hosting type here), I think it's be better to just link to the email servers doc, and maybe even this part of the doc: https://www.odoo.com/documentation/master/applications/general/email_communication/email_servers.html#manage-inbound-messages

   Your **email server** must be configured to be able to create an **email alias**. If it isn't
   configured yet, click :guilabel:`Choose or Configure Email Servers` and configure the mail server/your alias domain [ref or doc --> see my comment below] .

I'm not a 100% sure here since I don't know this topic very well but  could we say "Your **alias domain** must be configured xxx" ??
- If not: I suggest we keep use: :doc:`configure the mail server <link to the mail server doc>`
- If yes: I suggest we use: :ref:`configure your alias domain <ref to the part of the mail server doc where we explain email aliases>`

Reply:
FRGI prefers to use mail server, updated as per your suggestion.